.github/workflows/self-smoke-test-action.yml

```diff
@@ -42,13 +47,11 @@ jobs:
 
     steps:
     - name: Check out the action locally
-      uses: actions/checkout@v3
-      with:
-        path: test
+      uses: actions/checkout@v4
```

Comment:
Can we not make version upgrades in the same PR? And I'd rather not change the testing logic/dirs unless it's required somehow.

Reply:
Ok, I'll stick with `actions/checkout@v3` and keep the previous `test` path.